app/Helpers/StringHelpers.php

```diff
@@ -287,4 +287,15 @@ public static function summation(array $array = [])
                 return implode(', ', $array) . ', and ' . $last;
         }
     }
+
+    public static function pageBlades($page)
+    {
+        $firstWord = strtolower(strtok($page, ' '));
```

Comment:
I've not seen `strtok` used like this before; I think `explode` is usually used for this type of work:
```php
$firstWord = strtolower(explode(' ', $page)[0]);
```